Testing/k6/services/encounter/README.md

```diff
@@ -0,0 +1,14 @@
+# HG Patient Service Testing  of various test types.
```

Comment:
This seems to be copied from the readme for patient and the title hasn't been updated to describe encounters.

Similarly, /Testing/k6/services/immunization/README.md looks like it was copied from the readme for medications and the title hasn't been updated to describe immunizations.

Reply:
ooh good cactch will update.